fitz/utils.py

```diff
@@ -536,25 +536,36 @@ def get_text_words(
     flags: OptInt = None,
     textpage: TextPage = None,
     sort: bool = False,
+    delimiters=None,
 ) -> list:
     """Return the text words as a list with the bbox for each word.
 
     Args:
         flags: (int) control the amount of data parsed into the textpage.
+        delimiters: (str,list) characters to use as word delimiters
```

Comment:
delimiters are _extra_ characters - i think we always treat 32 and 160 as delimiters?

Reply:
Yes, read as *extra* delimiters